engine/src/engine.ts

```diff
@@ -105,41 +106,44 @@ export interface LogEntry {
 }
 
 const replaceRegex = new RegExp(
-  `\\b((${Command.Pass}|${Building.GaiaFormer}|${Command.UpgradeResearch}) ?([^. ]+)?)(\\.|$)`,
+  `\\b((${Command.Pass}|${Building.GaiaFormer}|${Command.FormFederation} [^ ]+|${Command.UpgradeResearch}) ?([^. ]+)?)(\\.|$)`,
```

Comment:
Like for the others - add a test to make sure that adding extra strings after `Command.FormFederation` doesn't mess up the replays